binstar_client/mixins/package.py

```diff
@@ -4,18 +4,31 @@
 @author: sean
 '''
 from binstar_client.utils import jencode
+from binstar_client.errors import Conflict
+
 
 class PackageMixin(object):
 
     def copy(self, owner, package, version, basename=None,
-                     to_owner=None, from_label='main', to_label='main'):
+             to_owner=None, from_label='main', to_label='main', replace=False, update=False):
         url = '%s/copy/package/%s/%s/%s' % (self.domain, owner, package, version)
         if basename:
             url += '/%s' % basename
 
         payload = dict(to_owner=to_owner, from_channel=from_label, to_channel=to_label)
         data, headers = jencode(payload)
-        res = self.session.post(url, data=data, headers=headers)
-        self._check_response(res)
+
+        if replace:
```

Comment:
What happens here, if you try to use update/replace on an non-existing file? Shouldn't we handle that?

Reply:
self._check_response(res) will raise an error and as I see it's a normal flow for anaconda-client